tox.ini

```diff
@@ -7,6 +7,7 @@ envlist =
 requires =
     setuptools >= 30.3.0
     pip >= 19.3.1
+    tox-pypi-filter >= 0.2
```

Comment:
Why? Isn't this installed by ci-helpers?

Reply:
I am on the fence with this. I have the same question as @Cadair but having this here is also a reminder that we pinned pytest. But having this here also means we gotta fix up the template and affiliated packages, no?

Reply:
Given the tox file doesn't actually use this I really don't think it belongs here.

Reply:
The issue is that if the version of pip or setuptools is too old, tox will create an isolated environment that has the requirements specified above. So the alternative to this fix is updating pip and setuptools in ci-helpers. Is that better?

Reply:
We certainly could/should update pip in ci-helpers 🤷‍♀

Reply:
@astrofrog I don't think I follow. Doesn't the script in ci-helpers install `tox-pypi-filter` before it runs tox anyway?

Reply:
Yes but if pip is not up to date, tox sets up a virtualenv in which to run itself  using the dependencies defined in ``requires``